common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -307,12 +319,73 @@ def test_time_allotted_field_is_visible_with_proctored_exam(self):
         self.course_outline.visit()
         self.course_outline.open_exam_settings_dialog()
 
+        # select addtional settings tab
+        self.course_outline.select_advanced_settings_tab()
+
         # When I select the proctored exams radio button
         self.course_outline.select_proctored_exam()
 
         # Then the time allotted text field becomes visible
         self.assertTrue(self.course_outline.time_allotted_field_visible())
 
+    def test_exam_review_rules_field_is_visible_with_proctored_exam(self):
+        """
+        Test that the review rules textarea field is shown
+        if proctored exam radio button is selected
+        """
+
+        # Given that I am a staff member
+        # And I have visited the course outline page in studio.
+        # And the subsection edit dialog is open
+        LogoutPage(self.browser).visit()
+        self._auto_auth("STAFF_TESTER", "staff101@example.com", True)
+        self.course_outline.visit()
+        self.course_outline.open_exam_settings_dialog()
+
+        # select addtional settings tab
+        self.course_outline.select_advanced_settings_tab()
+
+        # When I select the proctored exams radio button
+        self.course_outline.select_proctored_exam()
+
+        # Then the review rules textarea field becomes visible
+        self.assertTrue(self.course_outline.exam_review_rules_field_visible())
+
+    def test_exam_review_rules_field_is_not_visible_with_other_than_proctored_exam(self):
+        """
+        Test that the review rules textarea field is not
```

Comment:
Nit: we generally put the BDD description as the docstring of the test. That makes it easier to read what the test is intended to do, rather than having to read comments in the code.